src/environments/environment-config.interface.ts

```diff
@@ -5,4 +5,5 @@ export interface EnvironmentConfig {
   };
   appVersion: string;
   appRelease: string;
+  stage: string;
```

Comment:
Rather call this `stagePrefix`, because then you don't have an "empty" string in the default `environment`.

Also, make this optional like so: `stage?: string`, because then you don't have to specify an empty string if a stage does not have a prefix.